client/src/components/DeleteModal/DeleteSuccessModal.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
```

Comment:
Let's remove the `DeleteSuccessModal` - it's function is being replaced by `alertActions.show` (which is more consistent with the rest of the app). The only reason there was a `DeleteSuccessModal` in the first place is because it was written before the alert system was finished.